src/main/java/net/pms/util/SubtitleUtils.java

```diff
@@ -639,7 +639,7 @@ public static InputStream removeSubRipTags(File file) throws IOException {
 		BufferedReader input = FileUtil.bufferedReaderWithCorrectCharset(file);
 		ByteArrayOutputStream os = new ByteArrayOutputStream();
 		Writer writer = new OutputStreamWriter(os, Charset.forName(CHARSET_UTF_8));
-		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\[^\\}]+\\}|\\\\h|\\\\N");
+		Pattern pattern = Pattern.compile("\\</?(?:b|i|s|u|font[^\\>]*)\\>|\\{\\\\.*?}|\\\\h|\\\\N");
```

Comment:
I don't know if I agree that this make it simpler, but it makes is a few characters shorter :stuck_out_tongue:  I don't like to use the non-greedy unless I have to, I never really trust that it does what i want. Escaping the closing ```}``` is something I do because I think it makes reading the regex easier (it's obvious that it's a literal). But, the result should be the same :wink: